limiter/lifo_blocking.go

```diff
@@ -11,7 +11,7 @@ import (
 
 type lifoElement struct {
```

Comment:
These changes were included because I branched off https://github.com/platinummonkey/go-concurrency-limits/pull/78

If that PR gets merged I can rebase and this diff should be resolved